bulk_pull_account.py

```diff
@@ -207,9 +207,11 @@ def main():
 
 class TestBulkPullAccount(unittest.TestCase):
     def setUp(self) -> None:
+        # Good peer: ::ffff:94.130.12.236:7075
+
         self.ctx = livectx
-        peer = get_random_peer(self.ctx, lambda p: p.score >= 1000)
-        peeraddr, peerport = str(peer.ip), peer.port
+        peeraddr = '::ffff:94.130.12.236'
+        peerport = 7075
         self.s = get_connected_socket_endpoint(peeraddr, peerport)
         self.account = binascii.unhexlify(self.ctx['genesis_pub'])
         self.hdr = message_header(network_id(67), [18, 18, 18], message_type(11), 0)
```

Comment:
Do not hardcode network_id(67), use a ctx object.
Do not use a number for message_type, use a name.